fedimint-testing/src/gateway.rs

```diff
@@ -163,8 +165,17 @@ impl GatewayTest {
         }
     }
 
-    pub async fn wait_for_webserver(api: SafeUrl, password: Option<String>) -> anyhow::Result<()> {
-        let rpc = GatewayRpcClient::new(api, password);
+    /// Asynchronously waits for the webserver to be ready.
```

Comment:
I think it's best to skip "Asynchronously" here. The function is async, but it just "blocks" from the code's PoV.